packages/project-access/src/constants.ts

```diff
@@ -8,5 +8,6 @@ export const FileName = {
     Ui5Yaml: 'ui5.yaml',
     Ui5LocalYaml: 'ui5-local.yaml',
     Ui5MockYaml: 'ui5-mock.yaml',
-    UI5DeployYaml: 'ui5-deploy.yaml'
+    UI5DeployYaml: 'ui5-deploy.yaml',
+    MtaYaml: 'mta.yaml'
```

Comment:
Not want to be too picky, but the names are (mostly) in alphabetic order, would you mind move it below `ManifestAppDescrVar`.

Reply:
Updated in alphabetic order